### DISCUSSION TITLE
antd tree整行控制

### DISCUSSION THREAD

**Discussion by szpilm** [ant-design/ant-design#50290] [General]
我想antd tree整行控制样式，接收外部拖拽事件，但目前无法实现，只能通过title接收事件，控制样式

**Comment by SINK-MASTER** [comment]
参考官网示例
https://ant-design.antgroup.com/components/tree-cn#components-tree-demo-draggable
https://ant-design.antgroup.com/components/tree-cn#components-tree-demo-block-node

预览地址
https://jhgxcm.csb.app/

Demo
https://codesandbox.io/p/sandbox/ant-tree-jhgxcm
